dev/com.ibm.ws.install.utility.installUtility/resources/com/ibm/ws/install/internal/resources/MapBasedInstallKernelMessages.nlsprops

```diff
@@ -0,0 +1,31 @@
+# IBM Confidential OCO Source Material
+# (C) COPYRIGHT International Business Machines Corp. 2015.
+# The source code for this program is not published or otherwise divested
+# of its trade secrets, irrespective of what has been deposited with the
+# U.S. Copyright Office.
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME com.ibm.ws.install/resources/com/ibm/ws/install/resources/MapBasedInstallKernelMessages.nlsprops
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+# NLS_MESSAGEFORMAT_VAR
+# Messages in the range CWWKF1650 to 1699 are reserved for the asset manager.
+
```

Comment:
Suggested changes for all messages in com.ibm.ws.install/resources/com/ibm/ws/install/resources/MapBasedInstallKernelMessages.nlsprops:

-- Use "Liberty" instead of "Liberty profile" (remove "profile")
-- Use the correct URL to the support site. Someone on WAS support can give you the correct URL. Maybe https://www.ibm.com/support/home/ or 
https://www.ibm.com/support/home/product/C578916B44100K52/WebSphere_Application_Server